hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconStorageContainerManagerFacade.java

```diff
@@ -92,14 +95,22 @@ public ReconStorageContainerManagerFacade(OzoneConfiguration conf,
     this.ozoneConfiguration = getReconScmConfiguration(conf);
     this.scmStorageConfig = new ReconStorageConfig(conf);
     this.clusterMap = new NetworkTopologyImpl(conf);
+    DBStore dbStore = new SCMDBDefinition().createDBStore(conf);
```

Comment:
Should this not be "new ReconDBDefinition()"?

Reply:
Good catch, thanks. Fixed.